.github/workflows/release-samples.yml

```diff
@@ -1,10 +1,9 @@
-name: Upload Release Asset
+name: Upload Release Assets
 
 on:
   push:
-    # Sequence of patterns matched against refs/tags
-    tags:
-    - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
```

Comment:
I think we still need to have a tag for the push or later on `github.ref` will resolve to the name of the branch (I think). It would always be something like `refs/heads/main` without a tag. I'm not 100% sure of that though...

Reply:
I'm not sure I understand. The purpose of this is to check the name of releases and if it matches 'v*' then it runs this action no? Instead of running it on releases we are now running it on every push to master.

Reply:
The issue is that the name of the release comes from `github.ref`, but if there is no tag, what does the name of the release become? Looking at this doc (https://docs.github.com/en/actions/learn-github-actions/contexts), it appears as though the default value would be `refs/heads/main` which isn't a terribly useful release name.

Reply:
Ok I see your point. I've changed the name of the release to be equal to the commit sha

Reply:
@kesmit13 good to go?